src/widgets/cell_detail.ts

```diff
@@ -116,13 +117,34 @@ export class CellDetailDialog extends Overlay {
       }
     });
 
-    if (tag) {
+    if (tags && tags.length > 0) {
       this.title.innerText = 'Cell Identification';
-      this.infoField =
-          this.insertField({content: tag, type: 'textarea', disabled: true});
-      this.infoField.classList.add('rounded-input', 'large');
+
+      this.description = document.createElement('table');
+      /*let rows =
+          [{user_name: 'Marked by', tag: 'Cell Identification'}, ...tags];*/
+      let rows =
+          [{created: 'Created', tag: 'Label', user_name: 'Marked by'}, ...tags];
+      let thead = document.createElement('thead');
+      let tbody = document.createElement('tbody');
+      rows.forEach((tag: any, i: number) => {
+        const row = document.createElement('tr');
+        const created = document.createElement(`${i ? 'td' : 'th'}`);
+        const user = document.createElement(`${i ? 'td' : 'th'}`);
+        const tagTD = document.createElement(`${i ? 'td' : 'th'}`);
+        created.innerText =
```

Comment:
this ends up saying "Invalid date" rather than "Created"

Reply:
fixed!